homeassistant/components/inels/translations/cs.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
Additional translations should be removed from the PR, they are automatically included at build-time.